pkg/daemon/pinned_image_set.go

```diff
@@ -240,6 +240,8 @@ func (p *PinnedImageSetManager) sync(key string) error {
 		return err
 	}
 
+	klog.Infof("All pinned image sets complete")
```

Comment:
(non-blocking for tech preview) Do you know if this log (and the status update below) is repeatedly called after a pinnedimageset completes, on any random sync?

So basically in steady state this would just log and the MCN gets an update (which probably isn't an issue but probably no need to add this log since I don't think it really helps us know what finished)

Reply:
agree will drop

Reply:
Dropped